### DISCUSSION TITLE
Create email templates with Gutenberg blocks

### DISCUSSION THREAD

**Discussion by takerumaeda** [hizzle-co/noptin#721] [Ideas]
It would be very useful to be able to create email templates with Gutenberg block.
The customizability of the styling is dramatically improved.

**Comment by picocodes** [comment]
Hi @takerumaeda,

Email clients don't support modern HTML and CSS hence we can't use the block editor that is bundled with WordPress.

We've, however, released a custom block editor that you can use to style and customize your emails.

Update both Noptin and the Addons Pack then let me know if we missed anything or if you notice any bugs.
